astro/src/content/docs/get-started/start-here/step-3.mdx

```diff
@@ -0,0 +1,78 @@
+navOrder: 0
+codeRoot: https://raw.githubusercontent.com/fusionauth/fusionauth-example-get-started/main
+---
+
+<div className="border border-green-700 px-3 rounded-md">
+  <i className="fa-duotone fa-stopwatch fa-2x mr-2 text-green-700 align-middle"/> Estimated time to complete this step: **5 minutes**
+</div>
+
+## Protecting pages
+
+Now that we have the FusionAuth and the example app running, let's take a look at how the application protects pages that require a user to be logged in.
+
+Open the file `src/index.mts` and find the route named `/account`. It should look something like this:
+
+```typescript
+app.get("/account", async (req, res) => {
+  if (!await sdk.userHasAccess(req, res, ["admin", "user"])) {
+    res.redirect(302, '/login');
+    return;
+  }
+
+  res.sendFile(path.join(__dirname, '../templates/account.html'));
+});
+```
+
+The first check is where the application determines if the user is logged in or not. This is done by calling the SDK function `userHasAccess` (you can ignore the last parameter, which we will cover in the next section). This function returns true if the user has access, which implicitly verifies that they are logged in. Let's take a look at this function.
+
+Open the file `src/sdk.ts` and find the function named `userHasAccess`. One note is that this application app uses `jose` which is a Node library used for validating and parsing JWT. If you want to learn more about `jose`, you can visit the project at Github here: https://github.com/panva/jose
+
+The `userHasAccess` method begins by loading the user from the request. This is accomplished by calling the `getUser` function of the SDK. If the user comes back `null`, then this function returns false.
+
+```typescript
+async userHasAccess(req: Request, res: Response, roles: Array<string>): Promise<boolean> {
+  const jwt = await this.getUser(req, res);
+
+  // ...
+}
+```
+
+Let's take a quick peek at the `getUser` function of the SDK. This function looks like this:
+
+```typescript
+let accessToken = req.cookies[this.configuration.accessTokenCookieName];
+if (!accessToken) {
+  return null;
+}
+
+let payload: JWTPayload;
+try {
+  payload = (await jose.jwtVerify(accessToken, this.JWKS, {
+    issuer: this.configuration.oauthIssuer,
+    audience: this.configuration.applicationId,
+  })).payload;
+} catch (e) {
+  // ...
+}
+```
+
+This function loads a cookie from the request that contains the user's access token. The access token is part of the OAuth specification and FusionAuth access tokens are also JWTs. The second part of this function verifies that the JWT is valid using `jose`'s `jwtVerify` function. This function validates the cryptographic signature of the JWT and also ensures that the JWT is not expired and all of the claims are valid.
+
+We won't go into detail about JWTs and all of the claims here, but you can read our [OAuth token documentation](http://localhost:3000/docs/lifecycle/authenticate-users/oauth/tokens) to learn more.
```

Comment:
It might be helpful to have and `Aside` that's like "What's in a JWT?" on this page for people who may not have come across them. It could just be a one-line description like, "A FusionAuth JWT holds verifiable information about a user. It's digitally signed to ensure that the contents are authentic. For more information... "